src/components/user/LoginForm/styles.js

```diff
@@ -9,44 +10,44 @@ const styles = StyleSheet.create({
   login: {
     alignItems: 'center',
     flex: 1,
+    marginTop: 19,
     top: 200,
-    width: 210
+    width: wp('80%')
```

Comment:
Huh, never used this before, have you tried the app on different screen sizes?

Reply:
yes, it works fine!